app/controllers/application_controller.rb

```diff
@@ -69,7 +69,11 @@ def render_unauthorized
   end
 
   def saml_settings
-    SAML::SettingsService.instance.saml_settings
+    settings = SAML::SettingsService.instance.saml_settings
+    # TODO: 'level' should be its own class with proper validation
+    level = Integer(params[:level]) unless params[:level].nil?
```

Comment:
Functionally the same but I've usually seen this written as `params[:level].to_i`